next/components/forms/segments/AccountMarkdown/AccountMarkdown.tsx

```diff
@@ -52,7 +58,10 @@ const AccountMarkdown = ({ className, content, variant = 'normal' }: AccountMark
           a: ({ href, children }: { href?: string; children?: string }) => (
             <UILink
               href={href ?? '#'}
-              className="break-words font-semibold text-font underline hover:text-category-600"
+              className={cx('break-words font-semibold  underline hover:text-category-600', {
+                'text-white': uLinkVariant === 'primary',
+                'text-font': uLinkVariant === 'default',
+              })}
```

Comment:
I'd say this is again one of those where it's inconsistent in the design and it should get cleaned up, instead of adding variants to the code